core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -175,8 +181,68 @@ class WorldScreen(val gameInfo: GameInfo, val viewingCiv:CivilizationInfo) : Cam
     }
 
     private fun addKeyboardPresses() {
+        val quickSave = {
+            val toast = ToastPopup("Quicksaving...", this)
+            thread(name = "SaveGame") {
+                GameSaver.saveGame(gameInfo, "QuickSave") {
+                    Gdx.app.postRunnable {
+                        toast.close()
+                        if (it != null)
+                            ToastPopup("Quicksave failed!", this)
+                        else {
+                            ToastPopup("Quicksave OK.", this)
```

Comment:
We already have "failed to save game" and "Game saved" strings, try to reuse them to minimize translation additions

Reply:
I actually replaced the one with existing translations that would have been adequate - for consistency...

Reply:
"failed to save game" and "Game saved" do not exist, only "Could not save game!" -> perhaps just tone down the verbosity and omit what does not already have a translation?

_no_. This _needs_ feedback while existing save/load code is self-evident. Well then, equalize fail messages and today I dislike the "OK" - done.